src/storage/btree.c

```diff
@@ -8657,6 +8645,10 @@ btree_get_subtree_capacity (THREAD_ENTRY * thread_p, BTID_INT * btid, PAGE_PTR p
 	  ovfl_vpid = leaf_pnt.ovfl;
 	  if (!VPID_ISNULL (&ovfl_vpid))
 	    {			/* overflow pages exist */
+	      oid_cnt_ovfl = 0;
```

Comment:
The varibles as below looks better to be defined at this point because the variables is not used outside.
      int free_space_ovfl = 0;
      int oid_cnt_ovfl = 0;
      int pg_cnt_per_key = 0;

Reply:
OK, I will fix  it.